wpninjas/ninja-forms-conditional-logic
# Checkboxes will now show and hide fields when used in conditional statements

Closes #217 

Checkboxes will now show and hide fields when used in conditional statements.

Testing Steps:
Show and hide a field based on weather a checkbox is check or unchecked on the develop branch. It will not work. Switch to this branch and it should work properly.
